### ISSUE TITLE
Error "fs.readFileSync is not a function" when building nuxt

### ISSUE THREAD

**Issue by seybsen** [pimlie/nuxt-rfg-icon#10]
Using nuxt@2.3.4 with nuxt-rfg-icon@0.6.0 without any icons in `static/icons/` throws following error:

```
[ERROR] error while communicating with rfg api fs.readFileSync is not a function
  at Object.exports.fileToBase64Sync (eval at <anonymous> (node_modules/nuxt-rfg-icon/lib/utils.js:8:21), <anonymous>:26:15)
  at Object.exports.normalizeMasterPicture (eval at <anonymous> (node_modules/nuxt-rfg-icon/lib/utils.js:8:21), <anonymous>:197:45)
  at Object.exports.createRequest (eval at <anonymous> (node_modules/nuxt-rfg-icon/lib/utils.js:8:21), <anonymous>:238:38)
  at rfgApiRequestMeta (node_modules/nuxt-rfg-icon/lib/utils.js:113:23)
  at nuxt.hook (node_modules/nuxt-rfg-icon/lib/module.js:122:40)
  at fn (node_modules/@nuxt/common/dist/common.js:1720:47)
  at promise.then (node_modules/@nuxt/common/dist/common.js:1321:43)
  at <anonymous>
  at process._tickDomainCallback (internal/process/next_tick.js:228:7)
  at Function.Module.runMain (module.js:695:11)
  at startup (bootstrap_node.js:188:16)
  at bootstrap_node.js:609:3
```
I tracked down the error to this line, where all require's are replaced with an empty object. Unfortunately `var fs = {};` does not expose a function called `readFileSync()` which is needed later on in `fileToBase64Sync()`:
https://github.com/pimlie/nuxt-rfg-icon/blob/6146cd9976244e8edabba898278ef19fdeb3195d/lib/utils.js#L5

`init` holds this value:

``` var exports = {};
var https = {};
var fs = {};
var unzip = {};
var metaparser = {};
var fstream = {};
var mkdirp = {};
var axios = {};

/* ... */

exports.fileToBase64Sync = function(file) {
    return fs.readFileSync(file, { encoding: null }).toString('base64'); // ERROR
};
```
Build succeeds without the replace of requires in that line: `const init = rfgInit.init.toString();`, but I don't know what your comment means:

    // remove require's, we dont need them and the dependencies are bugged

**Comment by pimlie** [comment]
The reason for that ugly piece of code is because the unzip2 dependency which rfg-api uses doesnt work on v11 and latest v10 anymore (see https://github.com/RealFaviconGenerator/rfg-api/issues/16). If you include that dep node will give an error, so we have to wait before a new rfg-api release to fix that. In the mean time I implemented this trick so we dont have to wait for that. (I found it was too much work to re-implementing their logic to create a request for their api)

Unfortunately it seems that jest doesnt like an eval which also does a require, so the tests are currently broken. The fix does work for me locally, let me know if v0.6.1 works for you too.
